_posts/2017-03-08-mongolite.md

```diff
@@ -0,0 +1,79 @@
+---
+name: mongolite
+layout: post
+title: "Release mongolite 1.0"
+date: 2017-03-08
+authors:
+  - name: Jeroen Ooms
+categories:
+  - blog
+tags:
+- r
+- ropensci
+- packages
+- databases
+---
+
+
+After 2 years of development, version 1.0 of the [mongolite](https://cran.r-project.org/web/packages/mongolite/index.html) package has been released to CRAN. The package is now stable, well documented, and will soon be submitted for peer review to be onboarded in the ropensci suite.
+
+## MongoDB in R and mongolite
+
+I started working on mongolite in September 2014, and it was first announced at the ropensci [unconf 2015](https://twitter.com/_inundata/status/581605601882480640/photo/1). At this time, there were already two Mongo clients on CRAN: [rmongodb](https://cran.r-project.org/web/packages/rmongodb/index.html) (no longer works) and [RMongo](https://cran.r-project.org/web/packages/RMongo/index.html) (depends on Java). However I found both of them pretty clunky so I needed something better.
+
+Mongolite aims to provide a *simple* reliable R client for MongoDB, based on the excellent [mongo-c-driver](https://github.com/mongodb/mongo-c-driver) combined with super-powers from the [jsonlite](https://cran.r-project.org/web/packages/jsonlite/index.html) package. Simple means insert and query data in R using data-frames with a single command:
+
+```r
+# Make a connection
+con <- mongolite::mongo("diamonds", 
+  url = "mongodb://readwrite:test@ds043942.mongolab.com:43942/jeroen_test")
+
+# Find diamonds with: cut == Premium & price < 500
+mydata <- con$find('{"cut" : "Premium", "price" : { "$lt" : 500 } }')
+print(mydata)
+```
+
+In my experience, a simple interface is critical to get started. Obviously more advanced interfaces are available as well, but this will gets you up to speed right way if you just need the data and get on with your job.
```

Comment:
@jeroenooms i assume in _more advanced interfaces are available as well_ you mean in `mongolite`, yes? maybe say that if so